### PR TITLE
MFW-5962: Added dns filter license

### PR DESCRIPTION
**Pull request by sumedha-game**

### CODE REVIEW

File: services/licensemanager/testdata/licenses/licenses.json [hunk 1]
```diff
@@ -111,7 +124,8 @@
             "key": "90ba3b5264b10af43a205b7493118b3b",    
             "keyVersion": 3,                              
             "name": "untangle-node-dynamic-lists",            
-            "javaClass": "com.untangle.uvm.node.License"  
+            "javaClass": "com.untangle.uvm.node.License",
+            "valid": false
```
Comment (avinashdige15) [inline services/licensemanager/testdata/licenses/licenses.json:128]: Is this change required for dynamic-lists?

**Review by acooke-arista — APPROVED** [review]
